recipes/repo_hosting.rb

```diff
@@ -0,0 +1,107 @@
+#
+# Cookbook Name:: phabricator
+# Recipe:: repo_hosting
```

Comment:
Actually, I'd suggest not to use abbreviations unless they are really long (like VCS). `repo` could be replaced with `repository`.